charts/cloudnative-pg/templates/_helpers.tpl

```diff
@@ -36,9 +36,7 @@ Common labels
 {{- define "cloudnative-pg.labels" -}}
 helm.sh/chart: {{ include "cloudnative-pg.chart" . }}
 {{ include "cloudnative-pg.selectorLabels" . }}
-{{- if .Chart.AppVersion }}
```

Comment:
dropped this if conditional seeing as it appears to not be necessary anymore - unless I'm missing something non-obvious.